python/fledge/services/core/api/health.py

```diff
@@ -0,0 +1,126 @@
+                "commonInsert": 30,
+                "commonSimpleQuery": 3,
+                "commonQuery": 91,
+                "commonUpdate": 2,
+                "commonDelete": 1,
+                "readingAppend": 0,
+                "readingFetch": 0,
+                "readingQuery": 1,
+                "readingPurge": 0
+              },
+              "disk": {
+                "used": 95287524,
+                "usage": 82,
+                "available": 20918108,
+                "status": "green"
+              }
+           }
+
+    :Example:
+           curl -X GET http://localhost:8081/fledge/health/storage
+    """
+
+    # We need to find the address and management host for the Storage service.
+    from fledge.services.core.service_registry.service_registry import ServiceRegistry
+    from fledge.services.core.service_registry.exceptions import DoesNotExist
+    try:
+        services = ServiceRegistry.get(name="Fledge Storage")
+        service = services[0]
+    except DoesNotExist:
+        _LOGGER.error("Cannot ping the storage service. It does not exist in service registry.")
+        raise web.HTTPNotFound(reason="Cannot ping the storage service. It does not exist in service registry.")
+    try:
+        from fledge.common.service_record import ServiceRecord
+        if service._status != ServiceRecord.Status.Running:
+            msg = "The Storage service is not in Running state."
+            raise web.HTTPInternalServerError(reason=msg, body=json.dumps({"message": msg}))
+
+        from fledge.common.microservice_management_client.microservice_management_client import \
+            MicroserviceManagementClient
+
+        mgt_client = MicroserviceManagementClient(service._address,
+                                                  service._management_port)
+        ping_response = await mgt_client.ping_service()
+
+    except Exception as ex:
+        msg = str(ex)
+        _LOGGER.error("Could not ping Storage  due to {}".format(msg))
+        raise web.HTTPInternalServerError(reason=msg, body=json.dumps({"message": msg}))
+
+    try:
+
+        ping_response['disk'] = {}
+        disk_check_process = await asyncio.create_subprocess_shell('df -k ' + _FLEDGE_DATA,
+                                                                   stdout=asyncio.subprocess.PIPE,
+                                                                   stderr=asyncio.subprocess.PIPE)
+
+        stdout, stderr = await disk_check_process.communicate()
+        if disk_check_process.returncode != 0:
+            msg = "Could get disk stats due to {}".format(str(stderr))
+            _LOGGER.error(msg)
+            raise web.HTTPInternalServerError(reason=msg, body=json.dumps({"message": msg}))
+
+        disk_stats = stdout.decode("utf-8")
+        required_stats = disk_stats.split('\n')[1].split()
+        used = int(required_stats[2])
+        available = int(required_stats[3])
+        usage = int(required_stats[4].replace("%", ''))
```

Comment:
add comments which exact headers are you using? I assume order is always the same on different distributions.